ui/nginx.prod.conf

```diff
@@ -1,6 +1,9 @@
 # nginx server for static assets. For directive documentation, see
 # http://nginx.org/en/docs/dirindex.html
 
+# Note: this nginx configuration is designed for Google App Engine, to deploy Job Manager on other type of servers,
```

Comment:
Thanks for raising this. It's news to me that you would need to override this file. Can you add more detail about what the issue was using this for Kubernetes? Also, if there's another nginx conf file you're using we should consider adding it here or reconciling it with this one.

Reply:
We are currently using this nginx config: https://github.com/HumanCellAtlas/secondary-analysis/blob/master/deploy/job-manager/nginx.conf.ctmpl All we have to do is creating a configMap object and mount it to the path `/etc/nginx/nginx.conf`

There are not really many problems prevent us from us from using the nginx file here, except:
- k8s cannot find the path for these log dirs e.g. `/var/log/app_engine/app.log;`
- In order to make the GKE work, we have to configure the readinessProbe, so we added a hacky `/health` endpoint/location in nginx config.
- In some cases, we want to enable a HTTPBasicAuth proxy, so we have to make the switch for auth basic optional in nginx config.